src/main.ts

```diff
@@ -47,7 +48,17 @@ const command: Command = {
 		return createTask((callback: any) => rimraf(join('dist', 'src', name), callback))
 			.then(() => createChildProcess('tcm', [join('src', name, '*.m.css')], 'Failed to build CSS modules'))
 			.then(() =>
-				createChildProcess('tsc', ['--outDir', join('dist', 'src', name)], `Failed to build ${name}/index.d.ts`)
+				createTask((callback: any) =>
```

Comment:
Do you think we should clean this file up? It's unusual to have a nested tsconfig.json so might confuse users if they suddenly start to appear?

Reply:
That's what I was going to suggest as well. When I initially suggested this approach my assumption was that the generated tsconfigs would be put in a temp directory and then removed afterward.